pycbc/inference/sampler_kombine.py

```diff
@@ -213,16 +210,9 @@ def burn_in(self):
         else:
             blob0 = None
         res = self._sampler.burnin(self.p0, blob0=blob0)
+        # store the number of iterations used
+        self.burn_in_iterations = numpy.repeat(self.niterations, self.nwalkers)
```

Comment:
There's a lot of effort and extra layer of complexity for having different burn in iterations for different walkers. What's the use case for this?